integration_tests/ethereum/contracts/MockAaveV2StablecoinCellar.sol

```diff
@@ -1,59 +1,66 @@
 // SPDX-License-Identifier: Apache-2.0
 pragma solidity ^0.8.0;
 
-import "./interfaces.sol";
+import {ERC20, Ownable} from "./interfaces.sol";
 
 contract MockAaveV2StablecoinCellar is Ownable {
     /**
      * @notice Whether or not the contract is shutdown in case of an emergency.
      */
     bool public isShutdown;
 
-    event mockAccrueFees();
-    event mockTransferFees();
+    event mockAccrue();
+    event mockSendFees();
     event mockSetFeesDistributor(bytes32 newFeesDistributor);
     event mockEnterPosition();
+    event mockExitPosition();
     event mockRebalance(address[9] route, uint256[3][4] swapParams, uint256 minAssetsOut);
     event mockReinvest(uint256 minAssetsOut);
     event mockClaimAndUnstake();
-    event mockSweep(address token, address to);
+    event mockSweep(ERC20 token, address to);
     event mockSetLiquidityLimit(uint256 limit);
     event mockSetDepositLimit(uint256 limit);
-    event mockSetTrust(address position, bool trust);
-    event mockSetShutdown(bool shutdown, bool exitPosition);
+    event mockSetTrust(ERC20 position, bool trust);
+    event mockShutdownInitiated(bool emptyPosition);
+    event mockShutdownLifted();
 
     constructor() {}
 
     /**
-     * @notice Take platform fees and performance fees off of cellar's active assets.
+     * @notice Accrue yield, platform fees, and performance fees.
      */
-    function accrueFees() external {
-        emit mockAccrueFees();
+    function accrue() external {
+        emit mockAccrue();
     }
 
     /**
      * @notice Transfer accrued fees to the Sommelier Chain to distribute.
      */
-    function transferFees() external onlyOwner {
-        emit mockTransferFees();
+    function sendFees() external onlyOwner {
+        emit mockSendFees();
     }
 
     /**
-     * @notice Update the address of the fee distributor on the Sommelier Chain. IMPORTANT: Ensure
-     *         that the address is formatted in the specific way that the Gravity contract expects
-     *         it to be.
+     * @notice Set the address of the fee distributor on the Sommelier chain.
      */
     function setFeesDistributor(bytes32 newFeesDistributor) external onlyOwner {
         emit mockSetFeesDistributor(newFeesDistributor);
     }
 
     /**
-     * @notice Enters into the current Aave stablecoin strategy.
+     * @notice Pushes assets into the current Aave lending position.
      */
-    function enterPosition() external onlyOwner  {
+    function enterPosition() external whenNotShutdown onlyOwner  {
```

Comment:
`enterPosition` and `exitPosition` here still seem to be misisng their arguments.